### PR TITLE
added url oas_url filling on window load via url get param

### PR DESCRIPTION
**Pull request by XPOL555**
this would fix https://github.com/teamdigitale/api-oas-checker/issues/23

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -147,3 +147,9 @@ function selectivelyParse() {
 }
 editor.on('change', (event) => { selectivelyParse(); });
 
+window.addEventListener('load', function() {
+    pageUrl = new URL(window.location.href);
+    if(pageUrl.searchParams.get('url')){
+        document.getElementById('oas_url').value = pageUrl.searchParams.get('url');
+    }
```
Comment (ioggstream) [inline public/js/index.js:154]: Is that possible to trigger the validation without relying on the button?

**Comment by ioggstream** [comment]
@XPOL555 Thanks for your contribution! See the suggestion above :rocket:

File: public/js/index.js (hunk 1 above)
Comment (XPOL555) [reply public/js/index.js:154]: @ioggstream sure we just need to call `parseUrl()` after line 153 
and no - load will just wait the page to be loaded

Comment (XPOL555) [reply public/js/index.js:155]: i added the `parseUrl()` call

Comment (sebbalex) [reply public/js/index.js:155]: I know, I'm a nuisance, could we keep same lint rule (2 spaces) for this snippet?

Comment (ioggstream) [reply public/js/index.js:155]: I'll do it :P

File: public/js/index.js [hunk 2]
```diff
@@ -147,3 +147,10 @@ function selectivelyParse() {
 }
 editor.on('change', (event) => { selectivelyParse(); });
 
+window.addEventListener('load', function() {
+    pageUrl = new URL(window.location.href);
+    if(pageUrl.searchParams.get('url')){
+        document.getElementById('oas_url').value = pageUrl.searchParams.get('url');
+	parseUrl();
```
Comment (ioggstream) [inline public/js/index.js:154]:
```suggestion
        parseUrl();
```

File: public/js/index.js [hunk 3]
```diff
@@ -147,3 +147,10 @@ function selectivelyParse() {
 }
 editor.on('change', (event) => { selectivelyParse(); });
 
+window.addEventListener('load', function() {
+    pageUrl = new URL(window.location.href);
+    if(pageUrl.searchParams.get('url')){
+        document.getElementById('oas_url').value = pageUrl.searchParams.get('url');
+        parseUrl();
+    }
```
Comment (sebbalex) [inline public/js/index.js:155]:
```suggestion
window.addEventListener('load', function() {
  pageUrl = new URL(window.location.href);
  if(pageUrl.searchParams.get('url')){
      document.getElementById('oas_url').value = pageUrl.searchParams.get('url');
      parseUrl();
  }
```
😸

**Pull request merged**